### ISSUE TITLE
Dual Entrance/Exit

### ISSUE THREAD

**Issue by PFCKrutonium** [OpenRCT2/OpenRCT2#3119]
**OS: Win10** 
**Version:*0.0.4* 
**Commit:*ea05666* 

Dual Entrance/Exit

- [ ] Reproducable in RCT2 (vanilla)?
- [x] Multiplayer?

**Steps to reproduce:**
1. https://www.twitch.tv/poshtophat/v/53346229 Starting at 01:15:03 

[BrokenEnterance.sv6.txt](https://github.com/OpenRCT2/OpenRCT2/files/165893/BrokenEnterance.sv6.txt)

**Comment by IntelOrca** [comment]
Were you both trying to build that station at the same time?

**Comment by PFCKrutonium** [comment]
I was not, no.

**Comment by Margen67** [comment]
Were clearance checks disabled?
I seem to be able to randomly make multiple entrances/exits as shown in the video if I disable clearance checks, and place the entrance inside of the exit, or vice versa.
Only one of them actually works.
If I demolish the ride with the glitched entrances/exits they stay, but become even more glitchy.
When they're clicked on they show a bugged rollercoaster with similar attributes to the demolished ride.
You cannot rename, construct, close/open, etc or do anything to it.
They can only be removed with the Tile inspector unless you build another ride, and go into construction mode to remove them.

This also happens in singleplayer.

**Comment by Gymnasiast** [comment]
@Margen67 : No, what you describe is another issue (or rather the result of uncareful use of a hacking function).
